### ISSUE TITLE
Is it possible to add a spectator list?

### ISSUE THREAD

**Issue by Forsnoy** [TKazer/CS2_External#36]
Is it possible to add a spectator list?

**Comment by ghastlytim** [comment]
> Is it possible to add a spectator list?

yes infact an external method on how to do this has already been uploaded to unknown cheats as of yesterday

**Comment by Forsnoy** [comment]
> > Is it possible to add a spectator list?
> 
> yes infact an external method on how to do this has already been uploaded to unknown cheats as of yesterday

I searched and didn’t find it :( can you please give me a link? Thanks in advance.

**Comment by Fr0go1** [comment]
> > > Is it possible to add a spectator list?
> > 
> > 
> > yes infact an external method on how to do this has already been uploaded to unknown cheats as of yesterday
> 
> I searched and didn’t find it :( can you please give me a link? Thanks in advance.

https://www.unknowncheats.me/forum/counter-strike-2-a/595247-spectator-list.html

**Comment by urfavmonkeycheats** [comment]
yo did anyone know how to ?
